extension/wallet/src/lib.rs

```diff
@@ -84,65 +168,6 @@ pub fn setup() {
     handler.forget();
 }
 
-/// Generates 24 random seed words in english  
-#[wasm_bindgen]
-pub fn bip39_seed_words() -> Result<JsValue, JsValue> {
-    let mnemonic = wallet::bip39_seed_words(Language::English);
-    let words = mnemonic.phrase();
-
-    Ok(JsValue::from_str(words))
-}
-
-/// Create a new wallet from the given seed words (mnemonic) with the given name and password.
-///
-/// Fails if the seed words are invalid or if the wallet with this name already exists.
-/// The created wallet will be automatically loaded.
-#[wasm_bindgen]
-pub async fn create_new_bip39_wallet(
-    name: String,
-    seed_words: String,
-    password: String,
-) -> Result<JsValue, JsValue> {
-    let mnemonic = Mnemonic::new(seed_words, Language::English)
-        .map_err(|e| JsValue::from_str(format!("Could not parse seed words: {:?}", e).as_str()))?;
-
-    map_err_from_anyhow!(
-        wallet::create_from_bip39(name, mnemonic, password, &LOADED_WALLET).await
-    )?;
-
-    Ok(JsValue::null())
-}
-
-/// Load an existing wallet.
-///
-/// Fails if:
-///
-/// - the wallet does not exist
-/// - the password is wrong
-#[wasm_bindgen]
-pub async fn load_existing_wallet(name: String, password: String) -> Result<JsValue, JsValue> {
-    map_err_from_anyhow!(wallet::load_existing(name, password, &LOADED_WALLET).await)?;
-
-    Ok(JsValue::null())
-}
-
-/// Unload the currently loaded wallet.
-///
-/// Does nothing if currently no wallet is loaded.
-#[wasm_bindgen]
-pub async fn unload_current_wallet() {
-    wallet::unload_current(&LOADED_WALLET).await
-}
-
-/// Retrieve the status of the wallet with the given name.
-#[wasm_bindgen]
-pub async fn wallet_status(name: String) -> Result<JsValue, JsValue> {
-    let status = map_err_from_anyhow!(wallet::get_status(name, &LOADED_WALLET).await)?;
-    let status = map_err_from_anyhow!(JsValue::from_serde(&status))?;
-
-    Ok(status)
-}
-
```

Comment:
I know that it is a result of our iterations but it would be nice to keep the diff small where possible (also to make your own life easier in resolving merge conflicts).

From what I can see, this part of the code didn't actually functionally change, it was just moved into dedicated files.